From: Bastian Krause <bst@pengutronix.de>
Date: Wed, 14 Apr 2021 17:36:24 +0200
Subject: [PATCH] CMake: install D-Bus policy configuration to
 CMAKE_INSTALL_DATADIR

The D-BUS system bus policy config should reside in
CMAKE_INSTALL_DATADIR rather than in CMAKE_INSTALL_SYSCONFDIR.

See:

  https://gitlab.freedesktop.org/dbus/dbus/-/blob/ef55a3db0d8f17848f8a579092fb05900cc076f5/bus/CMakeLists.txt#L117

Forwarded: https://github.com/Igalia/cog/pull/296
Signed-off-by: Bastian Krause <bst@pengutronix.de>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index cb51087359fc..2fcd63845212 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -157,7 +157,7 @@ if (COG_DBUS_SYSTEM_BUS)
     configure_file(dbus/policy.conf.in ${COG_DEFAULT_APPID}.conf @ONLY)
     install(
         FILES ${CMAKE_CURRENT_BINARY_DIR}/${COG_DEFAULT_APPID}.conf
-        DESTINATION ${CMAKE_INSTALL_SYSCONFDIR}/dbus-1/system.d
+        DESTINATION ${CMAKE_INSTALL_DATADIR}/dbus-1/system.d
         COMPONENT "runtime"
     )
 
